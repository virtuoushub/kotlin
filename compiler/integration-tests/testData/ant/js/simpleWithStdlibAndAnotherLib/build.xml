<project name="Ant Task Test" default="build">
  <taskdef resource="org/jetbrains/jet/buildtools/ant/antlib.xml" classpath="${kotlin.lib}/kotlin-ant.jar"/>
  <property name="library.path" value="${kotlin.lib}/../../jslib-example.jar"/>
  <target name="build">
    <kotlin2js src="${test.data}/root1" output="${temp}/out.js" main="call">
      <library>
        <pathelement path="${library.path}"/>
      </library>
    </kotlin2js>
    <unzip src="${library.path}" dest="${temp}">
      <patternset>
        <include name="*.js"/>
      </patternset>
    </unzip>
  </target>
</project>
