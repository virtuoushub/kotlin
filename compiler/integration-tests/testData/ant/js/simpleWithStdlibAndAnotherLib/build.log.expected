OUT:
Buildfile: [TestData]/build.xml

build:
[kotlin2js] Compiling [[TestData]/root1] => [[Temp]/out.js]
    [unzip] Expanding: [KotlinProjectHome]/dist/jslib-example.jar into [Temp]

BUILD SUCCESSFUL
Total time: [time]

Return code: 0
